types/w3c-web-serial/index.d.ts

```diff
@@ -0,0 +1,112 @@
+// Type definitions for non-npm package w3c-web-serial-browser 1.0
```

Comment:
Name here should match directory name here

Reply:
dtslint lead me astray here.